samples/pass_through_processing/docker-compose.l4t.yml

```diff
@@ -30,12 +30,13 @@ services:
       - ..:/opt/savant/samples
     command: samples/pass_through_processing/module.yml
     environment:
+      - LOGLEVEL=info,savant_core::pipeline::stats=error
```

Comment:
Move loglevel of latency stats to `debug` and change `LOGLEVEL` to `info` in the docker compose files.